### ISSUE TITLE
Mandatory Day Field

### ISSUE THREAD

**Issue by jakeau** [wger-project/wger#1249]
No need to include mandatory day for workouts. Some people will just follow Push, Pull, Legs, repeat mentality and each day may end up being different.

**Comment by rolandgeider** [comment]
yeah, making the routines more flexible is definitely on the todo list (#848). Also not forcing the user to do exactly the exercises in the plan
